### ISSUE TITLE
Unexpected token import​​ with basic Vue setup using import in a .js file

### ISSUE THREAD

**Issue by bbugh** [wallabyjs/public#1757]
### Issue description or question

One of my tests is failing in Wallaby with `Unexpected token import​​`. It works fine running just jest by itself. I am using `transform-es2015-modules-commonjs` in babel so it works in jest, and expect that Wallaby would pick that up since I'm using the Vue babel configuration. My best guess is that wallaby doesn't run under the `test` environment, but that would be an odd surprise.

This is where the issue arises in Wallaby:

```javascript
// src/lib/filters.js, where the issue happens
import Vue from 'vue'
import startCase from 'lodash-es/startCase'  // You've got fail!
import toLower from 'lodash-es/toLower'
```

### Wallaby.js configuration file

Note this is exactly the Vue 2.x one from the documentation, just with files/tests changed.

```javascript
module.exports = function (wallaby) {
  return {
    files: ['app/javascript/**/*', '!app/javascript/**/*.spec.js', 'package.json'],

    tests: ['app/javascript/**/*.spec.js'],

    env: {
      type: 'node',
      runner: 'node'
    },

    compilers: {
      '**/*.js': wallaby.compilers.babel(),
      '**/*.vue': require('wallaby-vue-compiler')(wallaby.compilers.babel({}))
    },

    preprocessors: {
      '**/*.vue': file => require('vue-jest').process(file.content, file.path)
    },

    setup: function (wallaby) {
      const jestConfig = require('./package').jest || require('./test/unit/jest.conf')
      jestConfig.transform = {}
      wallaby.testFramework.configure(jestConfig)
    },

    testFramework: 'jest',

    debug: true
  }
}
```

### Additional Config Files

<details>

```javascript
// .babelrc
{
  "presets": [
    ["env", {
      "modules": false,
      "targets": {
        "browsers": "> 1%",
        "uglify": true
      },
      "useBuiltIns": true
    }]
  ],

  "plugins": [
    "transform-object-rest-spread",
    ["transform-class-properties", { "spec": true }]
  ],

  "env": {
    "test": {
      "plugins": ["transform-es2015-modules-commonjs"]
    }
  }
}
```

package.json for jest
```json
{
  "jest": {
    "roots": [
      "app/javascript/"
    ],
    "moduleDirectories": [
      "node_modules",
      "app/javascript"
    ],
    "moduleFileExtensions": [
      "js",
      "json",
      "vue"
    ],
    "testRegex": "(/test/.*|\\.(test|spec))\\.(ts|tsx|js)$",
    "transform": {
      "^.+\\.js$": "<rootDir>/node_modules/babel-jest",
      ".*\\.(vue)$": "<rootDir>/node_modules/vue-jest"
    },
    "transformIgnorePatterns": [
      "/node_modules/(?!lodash-es)"
    ],
    "moduleNameMapper": {
      "^#/(.*)$": "<rootDir>/app/javascript/$1"
    },
    "snapshotSerializers": [
      "<rootDir>/node_modules/jest-serializer-vue"
    ],
    "setupTestFrameworkScriptFile": "given2/setup",
    "setupFiles": [
      "jest-plugin-context/setup"
    ]
  }
}
```
</details>

### Code editor or IDE name and version

Visual Studio Code v1.25.1

### OS name and version

OSX 10.12.6

**Comment by ArtemGovorov** [comment]
Looks like your `files` list doesn't contain any files/patterns from `src`, but as you've mentioned in your comment, the error happens in the `src/lib/filters.js` file. If it's the case, then adding your `src` folder pattern to your `files` list should help.

**Comment by bbugh** [comment]
Hi @ArtemGovorov, that's definitely not the issue. I used `src/lib/filters` as a placeholder directory name while I was writing the issue, and then removed all of the references (but left one by accident). The files are correctly mapped. All of our tests work fine except this one file, which we had to configure Jest to use via babel and `transformIgnorePatterns`. 

This still seems like a wallaby configuration issue, related to either how Wallaby is loading babel or jest.

```
// app/javascript/lib/filters.js

// this does not work with Wallaby
import startCase from 'lodash-es/startCase'
import toLower from 'lodash-es/toLower'

// this works
import { startCase, toLower} from 'lodash-es'
```

However, that working code results in a larger webpack size because it's not tree shaking.
